Assets/Scripts/PanoramicSlowRotation.cs

```diff
@@ -0,0 +1,16 @@
+using UnityEngine;
+
+public class PanoramicSlowRotation : MonoBehaviour
+{
+    [SerializeField]
+    private float _xSpeed;
+    [SerializeField]
+    private float _ySpeed;
+    [SerializeField]
+    private float _zSpeed;
+
+    public void Update()
+    {
+        transform.Rotate(new Vector3(_xSpeed, _ySpeed, _zSpeed));
```

Comment:
same as MainMenu.cs. If it won't be used anywhere else, might couple it to the gameObject on which it operates